README.md

```diff
@@ -273,9 +282,150 @@ dependencies:
 
 ## Editor Integration
 
-Note to editor integrators: there is now a haskell-language-server-wrapper executable, which is installed alongside the haskell-language-server executable. When this is invoked in the project root directory, it attempts to work out the GHC version used in the project, and then launch the matching haskell-language-server executable.
+Note to editor integrators: there is a haskell-language-server-wrapper executable, which is installed alongside the haskell-language-server executable. When this is invoked in the project root directory, it attempts to work out the GHC version used in the project, and then launch the matching haskell-language-server executable.
 
-All of the editor integrations assume that you have already installed haskell-language-server (see above) and that the installation script put the haskell-language-server binary in your path (usually `~/.local/bin` or `~/.cabal/bin` on linux and macOS).
+All of the editor integrations assume that you have already installed haskell-language-server (see above) and that the installation script put the haskell-language-server and haskell-language-server-wrapper binaries in your path (usually `~/.local/bin` or `~/.cabal/bin` on linux and macOS, `%APPDATA%\local\bin` or `%APPDATA%\cabal\bin` in windows).
+
+### Using Haskell Language Server with VS Code
+
+Install from
+[the VSCode marketplace](https://marketplace.visualstudio.com/items?itemName=alanz.vscode-hie-server), or manually from the repository [vscode-hie-server](https://github.com/alanz/vscode-hie-server).
+
+Choose `haskell-language-server` in the extension setting `languageServerHaskell.hieVariant`.
+
+### Using Haskell Language Server with Sublime Text
+
+- Make sure haskell-language-server and haskell-language-server-wrapper are installed (see above) and they are in the global `$PATH`.
+- Install [LSP](https://packagecontrol.io/packages/LSP) using [Package Control](https://packagecontrol.io/)
+- From Sublime Text, press Command+Shift+P and search for Preferences: LSP Settings
+- Paste in these settings. Make sure to change the command path to your `hie`
+
+```json
+{
+"clients": {
+  "haskell-ide-engine": {
+    "command": ["haskell-language-server-wrapper", "--lsp"],
+    "scopes": ["source.haskell"],
+    "syntaxes": ["Packages/Haskell/Haskell.sublime-syntax"],
+    "languageId": "haskell",
+  },
+},
+}
+```
+
+Now open a Haskell project with Sublime Text. You should have these features available to you:
+
```

Comment:
An additional step is missing: Ctrl + Shift + P (or Cmd + Shift + P) / "LSP: Enable Language Server in Project"